src/main/java/com/exasol/ExaCompilationException.java

```diff
@@ -0,0 +1,13 @@
+package com.exasol;
+
+/**
+ * This exception indicates that the compilation of a script failed, or
+ * that the script api callback functions are not correctly implemented.
+ */
+public class ExaCompilationException extends Exception {
+    private static final long serialVersionUID = 1L;
+    public ExaCompilationException() { super(); }
```

Comment:
javadoc will probably demand javadoc comments for public methods.

Reply:
I wanted a 1:1 copy in this PR. I will fix code smells and JavaDoc in a later PR.